Update install-app-guard.md

bzimnisky opened the pull request:
Update to clarify the licensing requirements

bzimnisky committed:
Update install-app-guard.md

Update to clarify the licensing requirements

JohanFreelancer9 commented on microsoft-365/security/office-365-security/install-app-guard.md:
```diff
@@ -43,10 +43,13 @@ To learn more about Office update channels, see [Overview of update channels for
 
 > [!NOTE]
 > Microsoft 365 Apps for enterprise with shared computer activation or device-based licensing do not have access to Application Guard for Office.
+> Safe Documents licensing plans allow access to Application Gurad for Office. For more information, see [Safe Documents in Microsoft 365 E5/A5](https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/safe-docs?msclkid=279c718bd08811ec93a29310a7e2c785&view=o365-worldwide).
```
```suggestion
>
> Safe Documents licensing plans allow access to Application Guard for Office. For more information, see [Safe Documents in Microsoft 365 E5/A5](/microsoft-365/security/office-365-security/safe-docs).
```

bzimnisky replied:
Still needs the host part but agree that the query string can be removed

JohanFreelancer9 reviewed (changes requested):
Copy review - @bzimnisky, kindly commit the suggested changes. Many thanks.

bzimnisky committed:
Merge branch 'public' into patch-3

bzimnisky committed:
Update install-app-guard.md

Updated licensing requirements, removed query string from safe docs URI based on comments

denisebmsft committed:
Merge branch 'public' into patch-3

JohanFreelancer9 reviewed (changes requested):
@bzimnisky, kindly change the link to a relative link as previously requested. The link will not work on the preview page but it will work in the live document once it is merged. For confirmation, you can look at other links in this doc (and other Microsoft docs). Thanks.

bzimnisky committed:
Merge branch 'public' into patch-3

bzimnisky committed:
Update install-app-guard.md

Updated based on comments

JohanFreelancer9 commented on microsoft-365/security/office-365-security/install-app-guard.md:
```diff
@@ -43,10 +43,13 @@ To learn more about Office update channels, see [Overview of update channels for
 
 > [!NOTE]
 > Microsoft 365 Apps for enterprise with shared computer activation or device-based licensing do not have access to Application Guard for Office.
+> Safe Documents licensing plans allow access to Application Gurad for Office. For more information, see [Safe Documents in Microsoft 365 E5/A5](/microsoft-365/security/office-365-security/safe-docs).
```
```suggestion
>
> Safe Documents licensing plans allow access to Application Guard for Office. For more information, see [Safe Documents in Microsoft 365 E5/A5](/microsoft-365/security/office-365-security/safe-docs).
```

JohanFreelancer9 reviewed (changes requested):
@bzimnisky, kindly commit the final suggested change to fix the typo ("Garud" to "Guard"). The additional blank line is also needed to separate the two notes. Thanks.

bzimnisky committed:
Update microsoft-365/security/office-365-security/install-app-guard.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

bzimnisky committed:
Merge branch 'public' into patch-3

bzimnisky committed:
Merge branch 'public' into patch-3

yogkumgit commented:
@schmurky Please merge this PR. Thanks

chrisda committed:
Merge branch 'public' into patch-3

bzimnisky committed:
Merge branch 'public' into patch-3

bzimnisky committed:
Merge branch 'public' into patch-3

bzimnisky commented:
Anything preventing this from getting merged?

yogkumgit commented:
@schmurky Gentle reminder.

schmurky committed:
Merge branch 'public' into patch-3

bzimnisky commented:
> Kindly confirm if this edit is correct. If we don't want users to have to change passwords, don't we have to check the box instead?

Unclear what this comment refers to. This edit is specific to supported licensing plans for Office in Application Guard.

yogkumgit commented:
@schmurky

bzimnisky committed:
Merge branch 'public' into patch-3

bzimnisky commented:
Ping @schmurky

schmurky committed:
Merge branch 'public' into patch-3
